lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -789,6 +789,11 @@ module Resources
         'op': 'replace',
         'path': '/name/0/family',
         'value': 'Potter'
+      },
+      {
+        'path': '/name/0/prefix',
+        'op': 'replace',
+        'value': ['Mr.']
```

Comment:
Can the sequence be same as above, like,  op, path and then value?

Reply:
Updated here https://github.com/cerner/fhir.cerner.com/commit/1e0d55147568928481c491904b8ad3f917100689

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/4de910767a99e9946ef8e2d341b8325dd0757d41